_posts/alueker/2021-11-16-alueker-project-update-and-schedule.md

```diff
@@ -9,10 +9,10 @@ In my previous post I made a bold claim about the schedule of my game, a fully i
 I'm downscaling my project to one simple minigame, a functional version of the rhythm game "Bop It". For those who don't know, Bop It is/was a toy where players are given a series of commands that they have to follow in a sequential pattern. If the player is too slow or preforms an incorrect command, the game is over. On the physical game, there are 5 possible commands on the extreme edition: Bop It, Pull It, Twist It, Spin It, Flick It.
 In order to program the game into Python, there's a series of tasks that need to be completed for the project to be successful:
 
-- Align button commands (i.e. space bar for Bop, up key for Spin, down for Pull, etc.) with in-game commands
-- Create a timer that gradually degrades (gets faster as game progresses)
-- Create a score to count how many correct commands were performed in a row 
-- Create a score max that leads to a win screen
+1 Align button commands (i.e. space bar for Bop, up key for Spin, down for Pull, etc.) with in-game commands
+2 Create a timer that gradually degrades (gets faster as game progresses)
+3 Create a score to count how many correct commands were performed in a row 
+4 Create a score max that leads to a win screen
```

Comment:
For an ordered list in markdown, you'll want a `.` after the number:

1 this is just the numeral

1. This has a trailing period.

Reply:
Okay, fixed the periods, should be good to launch now.